modules/storages/app/views/storages/admin/storages/index.html.erb

```diff
@@ -1,31 +1,48 @@
 <!-- Standard Ruby view, please see the controller for comments -->
 <% html_title t(:label_administration), t("project_module_storages") %>
 
-<%= toolbar title: t("project_module_storages") do %>
-  <li class="toolbar-item">
-    <%= link_to new_admin_settings_storage_path,
-                { class: 'button -alt-highlight',
-                  aria: { label: t('storages.label_new_storage') },
-                  title: t('storages.label_new_storage') } do %>
-      <%= op_icon('button--icon icon-add') %>
-      <span class="button--text"><%= ::Storages::Storage.model_name.human %></span>
+<% if OpenProject::FeatureDecisions.storage_primer_design_active? %>
+  <%= render(Primer::OpenProject::PageHeader.new) do |component| %>
+    <% component.with_title(classes: "f1-light") do %>
```

Comment:
Hi @akabiru 
after longer discussion with the product team about the desired font size (and font weight), we agreed on 24px as default for the PageHeader Component. I just changed that in our viewComponents fork. Thus, the extra class here is not needed any more as soon you rebase.

Reply:
Thank you @HDinger , I'll rebase and update! 👍🏾